changes/test_tasks.py

```diff
@@ -311,6 +312,30 @@ def test_no_message_found(self, mock_create_outbound):
         )
         self.assertFalse(mock_create_outbound.called)
 
+    @mock.patch("changes.tasks.utils.ms_client.create_outbound")
+    @responses.activate
+    def test_timeout_message_sent_delivered(self, mock_create_outbound):
+        """
+        The task should send the correct outbound based on the delivered event.
+        """
+        self.create_outbound_lookup()
```

Comment:
We don't need to create the outbound lookup here, since we're mocking it out with the `@mock.patch`

Reply:
when I remove that line, I get a `requests.exceptions.ConnectionError: Connection refused: GET http://ms/api/v1/outbound/?vumi_message_id=messageid` error

Reply:
Oh right, sorry I got confused. The thing we're mocking out is the creation of a new message, and this adds the lookup for an existing message, they're two different things. We should then leave this here.